fix for "'gets' undeclared here" error with recent glibc
see also https://bugs.gentoo.org/424978
--- bison-2.5/lib/stdio.in.h.orig	2017-09-15 22:19:26.014249360 +0200
+++ bison-2.5/lib/stdio.in.h	2017-09-15 22:19:47.694669181 +0200
@@ -181,7 +181,6 @@
    so any use of gets warrants an unconditional warning.  Assume it is
    always declared, since it is required by C89.  */
 #undef gets
-_GL_WARN_ON_USE (gets, "gets is a security hole - use fgets instead");
 
 #if @GNULIB_FOPEN@
 # if @REPLACE_FOPEN@
